packages/contracts-bedrock/src/L1/OPPrestateUpdater.sol

```diff
@@ -0,0 +1,168 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+// Contracts
+import { OPContractsManager } from "src/L1/OPContractsManager.sol";
+
+// Interfaces
+import { ISuperchainConfig } from "interfaces/L1/ISuperchainConfig.sol";
+import { IProtocolVersions } from "interfaces/L1/IProtocolVersions.sol";
+import { IPermissionedDisputeGame } from "interfaces/dispute/IPermissionedDisputeGame.sol";
+import { ISystemConfig } from "interfaces/L1/ISystemConfig.sol";
+import { IDisputeGameFactory } from "interfaces/dispute/IDisputeGameFactory.sol";
+import { IFaultDisputeGame } from "interfaces/dispute/IFaultDisputeGame.sol";
+import { IDelayedWETH } from "interfaces/dispute/IDelayedWETH.sol";
+
+// Libraries
+import { Constants } from "src/libraries/Constants.sol";
+import { Claim, GameTypes } from "src/dispute/lib/Types.sol";
+
+/**
+ * @title OPPrestateUpdater
+ * @notice A custom implementation of OPContractsManager that allows for modified deployment parameters
+ */
+contract OPPrestateUpdater is OPContractsManager {
+    struct PrestateUpdateInput {
+        OpChain opChain;
+        Claim absolutePrestate;
```

Comment:
When you rebase, you should find that 
1. The `OpChain` struct is now `OpChainConfig`.
2. The prestate is contained within the `OpChainConfig` struct

Up to you if you even want to reuse the `OpChainConfig` struct or just  put all the values within this struct.

Reply:
Also, I'd recommend rebasing/merging sooner than later to avoid surprises.